tests/test/check/data/main.fmf

```diff
@@ -13,6 +13,8 @@
   /nasty:
     test: |
       set -x
+      systemctl status auditd || /bin/true
       sudo bash -c "passwd --help &> /root/passwd.log; \
                     ls -alZ /root/passwd.log; \
                     rm -f /root/passwd.log" || /bin/true
+      journalctl -b --no-pager
```

Comment:
This produces over 2000 lines of output. Do we really need the whole content? If so, what about storing it in a dedicated file under `$TMT_TEST_DATA` ? If not, what about using a more recent timestamp?

Reply:
Dropped in 579003ec

Reply:
Good!